code/modules/mob/dead/observer/observer.dm

```diff
@@ -145,26 +145,21 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	. = ..()
 	if(.)
 		return
-	switch(href_list[href])
-		if("reentercorpse")
+	if(usr == src)
```

Comment:
i am atleast 90% sure the switch is the issue

Reply:
and hell even if someone through some circumstance managed to execute `;join_larva=1;track_silo_number=5;` through some circumstance it would work AND even upstream TG does it